### DISCUSSION TITLE
Duplicate VMs and custom templates

### DISCUSSION THREAD

**Discussion by step76** [Moonif/MacBox#1] [Ideas]
Hello,

I think that MacBox is really great for macOS users!

Do you think is it possibile to improve the VMs managements adding VMs duplication function, maybe custom templates (could be a duplication extension for example to duplicate the configuration without storage).

And then would be nice to edit/rename the VM folder other than the title.

Thank you!

**Comment by Moonif** [comment]
Hello Step76,

Thank you for the suggestions! I'll try to add them in the upcoming update.
